content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -272,10 +270,11 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 
 ### Volume Mode
 
-Prior to v1.9, the default behavior for all volume plugins was to create a filesystem on the persistent volume. With v1.9, the user can specify a `volumeMode` which will now support raw block devices in addition to file systems. Valid values for `volumeMode` are "Filesystem" or "Block". If left unspecified, `volumeMode` defaults to "Filesystem" internally. This is an optional API parameter. 
+Prior to v1.9, the default behavior for all volume plugins was to create a filesystem on the persistent volume. With v1.9, the user can specify a `volumeMode` which will now support raw block devices in addition to file systems. Valid values for `volumeMode` are "Filesystem" or "Block". If left unspecified, `volumeMode` defaults to "Filesystem" internally. This is an optional API parameter.
+
 
 {{< note >}}
-**Note:** This feature is alpha in v1.9 and may change in the future. 
+**Note:** This feature is alpha in v1.9 and may change in the future. Alpha feature `BlockVolume` must be enabled in all cluster components for raw block storage to work.
```

Comment:
I think we use this tag now to indicate feature status:
```
{{< feature-state for_k8s_version="v1.9" state="alpha" >}}
```

Reply:
Fixed